package.json

```diff
@@ -18,8 +18,7 @@
         "Sublime Text"
     ],
     "activationEvents": [
-        "onStartupFinished",
-        "onCommand:extension.importFromSublime"
```

Comment:
We should change this to have a proper extension namespace?

Reply:
Yeah this extension generally needs some work to be a more modern extension